lib/rspec/rails/example/view_example_group.rb

```diff
@@ -161,6 +163,12 @@ def _include_controller_helpers
           controller.controller_path = _controller_path
           controller.request.path_parameters[:controller] = _controller_path
           controller.request.path_parameters[:action]     = _inferred_action unless _inferred_action =~ /^_/
+          controller.request.path = ViewPathBuilder.new(::Rails.application.routes).path_for(controller.request.path_parameters)
+          ViewSpecMethods.add_to(::ActionView::TestCase::TestController)
+        end
+
+        after do
+          ViewSpecMethods.remove_from(::ActionView::TestCase::TestController)
```

Comment:
Do we need these hooks to add the `extra_params`? I ask as there is access to [`params`](https://github.com/rspec/rspec-rails/blob/1fb6f9f642e887df1b1ee344594de02d7de3b3fe/lib/rspec/rails/example/view_example_group.rb#L77-L83) which is `public` in the view spec. We don't appear to have a feature scenario demoing it :grimacing: 

Would that be an alternative API to use? I may be missing some sort of glaring issue with how merging that with the path setup could cause issues.